### ISSUE TITLE
Black edged corner after minimizing windows in GNOME 41

### ISSUE THREAD

**Issue by MobCode100** [velitasali/GTKTitleBar#30]
Hi developer, just a simple/trivial bug but kinda annoying for me to look at

There's a black triangle at the corners right after minimizing window. Resizing the window will make it disappear. The bug only appears on some applications such as VSCode and VLC. Nautilus and terminal are fine.

![Screenshot from 2021-11-13 03-52-42](https://user-images.githubusercontent.com/66469454/141526707-b76c0d34-3013-4ec3-9615-d1d2aff1786a.png)

Video showing the problem: 
https://user-images.githubusercontent.com/66469454/141526186-3e3758d4-b8db-4869-8624-959543d69192.mp4

Using GNOME 41 on Fedora 35

Thank you! Any extra information, I'll gladly provide

**Comment by velitasali** [comment]
I am able to reproduce this. Thanks!

**Comment by cachandlerdev** [comment]
Has there been any progress on this issue? Unfortunately, it still seems to be present in GNOME 42 a year later. :(

**Comment by MobCode100** [comment]
> Has there been any progress on this issue? Unfortunately, it still seems to be present in GNOME 42 a year later. :(

Yep, the bug still present on Fedora 36 or you can try [other minimal extension](https://github.com/alecdotninja/no-titlebar-when-maximized) that doesn't have the issue.

**Comment by cachandlerdev** [comment]
I was using that one, but it seems to have performance issues when dragging windows around that this extension doesn't, so I was hoping that the corner problem might have been fixed. Thanks anyway!

**Comment by MobCode100** [comment]
> performance issues when dragging windows around

Wierd, I don't see any performance issues. Does it happen to specific software? Or is it in Wayland?

**Comment by cachandlerdev** [comment]
Oh, at first I though it was an extension problem after looking at the GitHub issues page, but it appears as though the lag was caused by a certain website (which makes sense, since the performance impact seemed to be a Wayland issue, and I was using Xorg). I suppose I'll continue using that extension for now.

**Comment by MobCode100** [comment]
Alright, thank you for your input ^^.
